code-challenges.test.js

```diff
@@ -54,3 +111,23 @@ const numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+// -->Psuedocode
+// Use let for three new arrays (array1, array2, and array3)
+// Tie each new array to the corresponding const and result.
+// Creat a for loop for each array with a starting index , array length using .length, and iterate over +1  
+
+// console log each result together or seperate
+
+
+
+ let array1 = numbersToAdd1, result1 = [];
+ let array2 = numbersToAdd2, result2 = [];
+ let array3 = numbersToAdd3, result3 = [];
+
+ for (let i = 0, s = array1[0]; i < array1.length; i++, s += array1[i]) result1.push(s);
+ for (let i = 0, s = array2[0]; i < array2.length; i++, s += array2[i]) result2.push(s);
+ for (let i = 0, s = array3[0]; i < array3.length; i++, s += array3[i]) result3.push(s);
+
```

Comment:
Would like to see this as a dynamic function next time.  Don't be afraid to use resources like google, the syllabus, and reaching out to us for help with syntax and logic.